packages/core/src/styles/style.scss

```diff
@@ -62,14 +62,19 @@ div.chart-holder {
 
 		path {
 			&.line {
-				fill: none;
 				stroke-width: 2;
 
 				&:hover {
 					cursor: default;
 				}
 			}
 		}
+		&:not(.area) path.line {
```

Comment:
is there anyway to do this using simpler selectors?

Reply:
Well, either we use this slightly more complex css to override svg element attributes, or we change all the uses of .attr("fill", [fill]) to .attr("style", [longer style stuff]) and then have to maintain more CSS in JS files.

Reply:
@theiliad what do you think about this? ☝️ Is it worth it?

Reply:
The current set of styles remove fills for all paths unless suggested otherwise.

Still trying to decide whether we should continue doing that or just specifically mention every line selector that shouldn't have a fill...

Reply:
resolved on Slack to use this selector